### ISSUE TITLE
API websocket connection fails due to CSP

### ISSUE THREAD

**Issue by tituspijean** [YunoHost-Apps/immich_ynh#138]
### Describe the bug

Server appears offline from the app.

Browser console regularly mentions:

> Content-Security-Policy : Les paramètres de la page ont empêché le chargement d’une ressource (connect-src) à l’adresse wss://domain.example/api/socket.io/?EIO=4&transport=websocket car elle enfreint la directive suivante : « default-src https: data: blob: »

### Context

YunoHost 12.0.7
Immich 1.120.2~ynh1

**Comment by Thovi98** [comment]
I've got a similar issue with Photoprism after activating exerimental features in Yunohost. Could you check if disabling them resolves your issue?

**Comment by tituspijean** [comment]
Yes, that's it! I actually wondered if I had changed anything in the last few days before I discovered the bug, and that's indeed the enabling of the experimental features. Thanks!

**Comment by tituspijean** [comment]
This is an instance of https://github.com/YunoHost/issues/issues/2126, let's close this issue for the time being.
